commands/train.py

```diff
@@ -85,6 +85,10 @@ def main(
 
 
     If there are multiple jobs in your projects, a model will be trained on each job.
+    KiliAUtoML will automatically select the right type of model according to your project
```

Comment:
Kili AutoML will automatically select the model types that match your Kili project's labeling tasks (...